packages/gestalt/src/SelectList/SelectList.css

```diff
@@ -11,7 +11,7 @@
   border-radius: 4px;
   border-style: solid;
   border-width: 1px;
-  padding: 11px 35px 11px 14px; /* right padding prevents it from running into arrow */
+  padding: 9.5px 35px 9.5px 14px; /* right padding prevents it from running into arrow */
```

Comment:
Fractional pixels here are going to cause the text to be blurry - perhaps you can remove the padding and fix the total `line-height` of the select list?

Reply:
It appears that using anything below line-height 17px results in the value/placeholder being cutoff. See: 
<img width="496" alt="screen shot 2018-03-06 at 3 29 11 pm" src="https://user-images.githubusercontent.com/12569707/37064782-26f7b672-2153-11e8-9fbb-104a0984be0d.png">
<img width="494" alt="screen shot 2018-03-06 at 3 29 04 pm" src="https://user-images.githubusercontent.com/12569707/37064783-270d41f4-2153-11e8-8542-b5e23fb2f9a6.png">

Reply:
I've used a mix of padding that isn't fractional and line height to achieve the 40px in the most recent commits.